open_spiel/higc/base64.cc

```diff
@@ -0,0 +1,132 @@
+/*
```

Comment:
There's some process around us importing third party code which I'd rather avoid.
Would you be able to use absl::Base64Escape instead?

Reply:
Yes, I updated the code to use absl implementation.